resources/js/components/dues/DuesRequiredInfo.vue

```diff
@@ -190,6 +190,26 @@ export default {
         .catch(error => {
           if (error && error.response && error.response.status === 400) {
             this.$emit('next');
+          } else if (
+            error &&
+            error.response &&
+            error.response.status === 422 &&
+            error.response.data &&
+            error.response.data.errors &&
+            typeof error.response.data.errors === 'object' && Object.keys(error.response.data.errors).length > 0 &&
+            typeof error.response.data.errors[Object.keys(error.response.data.errors)[0]] === 'object' &&
+            error.response.data.errors[Object.keys(error.response.data.errors)[0]].length > 0
+          ) {
+            const message = error.response.data.message;
+            const errors = error.response.data.errors;
+            let validation_messages = []
```

Comment:
I think this can be a `const` even if you add entries to the array (just JS things).  `let` allows the entire variable to be reassigned to a completely different value